TESTING_SETUP.md

```
@@ -0,0 +1,442 @@
+
+### High Coverage Modules (≥80%)
+- src/db/bakchod_dao.py: 100% (36/36 statements)
+- src/db/message_dao.py: 100% (11/11 statements)
+- src/db/quote.py: 100% (18/18 statements)
+- src/db/roll_dao.py: 100% (8/8 statements)
+- src/db/scheduledjob_dao.py: 100% (12/12 statements)
+- src/domain/analytics.py: 100% (4/4 statements)
+- src/domain/metrics.py: 100% (10/10 statements)
+- src/domain/rokda.py: 100% (6/6 statements)
+- src/domain/scheduler.py: 98% (53/54 statements)
+- src/db/__init__.py: 95% (74/78 statements)
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)